### DISCUSSION TITLE
Welcome to XmrSigner Discussions!

### DISCUSSION THREAD

**Discussion by XmrSigner** [XmrSigner/xmrsigner#3] [Announcements]
👋 Welcome to the Community!
We're building XmrSigner together as a community-driven project for air-gapped Monero transaction signing. This space is for:

🔧 Sharing build experiences and hardware tips
💡 Suggesting improvements and new features
🤝 Helping others with their XmrSigner builds
🛠️ Discussing development and contributions
🎨 Sharing case designs and modifications

Remember: This is a security-focused project, so:

Never share your private keys or seed phrases
Keep discussions focused on the public aspects of the project
Help maintain high signal-to-noise ratio in discussions

Get Started
Comment below with:

Your interest in XmrSigner
Any experience with similar projects
What you hope to contribute or learn

Together, we're making secure Monero transactions accessible to everyone! 💪

**Comment by Proskhmer08** [comment]
Can someone confirm that the SeedQR compact 25x25 works with the passphrase? I created the new seed using the xmr signer, then added a passphrase and then tried to export out the SeedQR. When I tried to scan it to verify it, I got errors. It could be possible that my manual copying of the SeedQR had an extra dot or missing dot, but I just wanted to make sure that it's been tested out successfully. 


<img width="1125" height="2436" alt="IMG_2794" src="https://github.com/user-attachments/assets/66dbf937-bf53-464d-a40b-441ef7380fba" />
<img width="1125" height="2436" alt="IMG_2795" src="https://github.com/user-attachments/assets/aa72fc6e-e2c5-4296-af2c-3c3759555f8b" />
<img width="1125" height="2436" alt="IMG_2796" src="https://github.com/user-attachments/assets/65daab3b-3dd7-4ee5-b33b-8eb89f835ce5" />
